### ISSUE TITLE
Give the moodbar/progress bar more space in a small window

### ISSUE THREAD

**Issue by paul4ig** [clementine-player/Clementine#5577]
![moodbar_space](https://cloud.githubusercontent.com/assets/6924479/21662696/39c96dec-d2d3-11e6-9248-575ca2dd8bd8.png)

**Comment by eturkes** [comment]
I agree, this is a bit frustrating, especially considering the moodbar is such a nice feature. [#2205](https://github.com/clementine-player/clementine/issues/2205) regarding blank space in the analyzer when it's turned off is also complimentary to this issue. It would neat if the icons and info to the left of the moodbar could be moved to the analyzer space.
